Sources/PayPalNativeCheckout/PayPalClient.swift

```diff
@@ -10,77 +8,71 @@ import PaymentsCore
 public class PayPalClient {
 
     public weak var delegate: PayPalDelegate?
-
+    /// instance to communicate with native checkout
```

Comment:
Nitpick - I think we can remove this comment, in favor of a docstring at the top of the `CheckoutProtocol`, or whatever name we land on.